ItaCH_Smash_Legends/Assets/Script/Util/Utils.cs

```diff
@@ -51,6 +51,8 @@ public static GameObject FindChild(GameObject go, string name = null, bool recur
         return targetTransform.gameObject;
     }
 
+    public static void ReverseAxisY(RectTransform rectTransform) => rectTransform.Rotate(0, 180, 0);
```

Comment:
```suggestion
    public static void FlipY(this RectTransform rectTransform) => rectTransform.Rotate(0, 180, 0);
```

확장 메소드로 사용하는 것이 더 좋아보입니다.